### PR TITLE
Manage SQL Information Protection Policy

### PR DESCRIPTION
**Pull request by bashahee**
## Description

Support management of SQL Information Protection Policy by adding two new Cmdlets: Set-AzSqlInformationProtectionPolicy and Get-AzSqlInformationProtectionPolicy.

The design was reviewed and accepted in https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/530.

Tests were added, and some existing tests records were updated.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by bashahee** [comment]
@cormacpayne @MiYanni @wyunchi-ms 
Hi all,
Is there any chance for this PR to be merged today, so it gets shipped with next upcoming release?

**Comment by cormacpayne** [comment]
@bashahee Hey Basel, @MiYanni and I are no longer a part of the Azure PowerShell team -- it looks like @msJinLei may be the on-call person for Azure PowerShell this release, so they should be able to help at their earliest convenience.

**Comment by bashahee** [comment]
@msJinLei Hi, please wait with your review, as I found a small bug in last minute.

**Comment by wyunchi-ms** [comment]
Hi @bashahee this PR can be merged today only if it can pass the CI.

**Comment by wyunchi-ms** [comment]
Hi @bashahee , is this PR ready for reviewing?

**Comment by bashahee** [comment]
@wyunchi-ms The PR is ready to be reviewed. However, it is not ready to be merged, I need to consume latest SDK clients package, after this PR is merged https://github.com/Azure/azure-rest-api-specs/pull/9053.

File: src/Security/Security/help/Set-AzSqlInformationProtectionPolicy.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+```
+
+## DESCRIPTION
+Sets the effective tenant SQL information protection policy.
+
+## EXAMPLES
+
+### Example
+```powershell
+PS C:\> Set-AzSqlInformationProtectionPolicy -Policy "C:\Users\myUser\Desktop\policy.json"
```
Comment (wyunchi-ms) [inline src/Security/Security/help/Set-AzSqlInformationProtectionPolicy.md:34]: This parameter should be `FilePath` not `Policy`.

**Comment by bashahee** [comment]
@wyunchi-ms Hi, the PR is ready to be merged now, thank you for the design and the PR review!

**Pull request merged**